x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_overview_logic.test.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { LogicMounter, mockHttpValues, mockFlashMessageHelpers } from '../../../__mocks__';
+
+import { nextTick } from '@kbn/test/jest';
+
+jest.mock('../engine', () => ({
+  EngineLogic: { values: { engineName: 'some-engine' } },
+}));
```

Comment:
Constance has a helper set up for this. Might make sense to use it here.

```
import { mockEngineValues } from '../../__mocks__';
```

```
beforeEach(() => {
    mockEngineValues.engineName = 'some-engine';
  });
```

Reply:
I've updated to include `import '../../__mocks__/engine_logic.mock';` instead in 18b0a91